server/index.js

```diff
@@ -91,6 +91,45 @@ app.post('/api/guests', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.put('/api/users/1', (req, res, next) => {
```

Comment:
Instead of hard-coding `1` into your url, you can hard-code it into your SQL. That way, down the road when you make the user id dynamic, you don't have to change your fetch or urls.